CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 This is only used for recording changes for major version bumps.
 More minor changes may optionally be recorded here too.
 
+## 56.0.0
+
+* `create_sanitised_html_for_url` must be imported from
+  `notifications_utils.markdown`, not `notifications_utils.formatters`
+* `MAGIC_SEQUENCE` and `magic_sequence_regex` must be imported from
```

Comment:
Does anything actually use any of these three values?

I can't find any references to them in Admin or API, so this seems like a minor change to me.

Reply:
I checked and yes, we don’t import any of these directly. Changed to a patch version bump instead but kept a CHANGELOG entry.